src/twisted/trial/test/test_assertions.py

```diff
@@ -749,7 +749,7 @@ def test_assertDictEqual(self):
         """
         self.assertDictEqual({'a': 1}, {'a': 1})
     if getattr(unittest.SynchronousTestCase, 'assertDictEqual', None) is None:
-        test_assertDictEqual.skip = (
+        test_assertDictEqual.skip = (  # type: ignore[attr-defined]
```

Comment:
Should this be using `@skipIf` ?

Reply:
I'm keeping that usage of `.skip` in that one place, because although I changed to use `@skipIf` in many places,  I want to make sure that the old behavior in trial which looks for a `.skip` attribute is not broken.